docs/install/any-kubernetes-cluster.md

```diff
@@ -24,25 +24,25 @@ You can install Knative by applying YAML files using the `kubectl` CLI.
 
 ## Prerequisites
 
-- You have a cluster that uses Kubernetes v1.17 or newer.
+- You have a cluster that uses Kubernetes v1.18 or newer.
 - You have installed the [`kubectl` CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 - If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
 - If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
+<!--TODO: Verify these requirements-->
+- Your Kubernetes cluster must have access to the internet, since Kubernetes needs to be able to fetch images.
 
 ## Installing the Serving component
 
-1. Install the
-   [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-   (aka CRDs):
+1. Install the required custom resources:
```

Comment:
I think you still need the word 'definitions' here to distinguish from an instance of the CRD, but I may well be being too pedantic and noone cares 😅.

```suggestion
1. Install the required custom resource definitions:
```

Reply:
I wondered this, however, in the kube docs, they seem to use CustomResourceDefinitions more specifically when talking about the API, and custom resources more as the thing that's actually installed?
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

I've been trying to nail down for a while when we should refer to what thing. If you think this is important to say though I'm OK with that, I think in general though we maybe need to find a way to make it clearer across docs the difference and when we mean CRs vs CRDs?

Reply:
I guess maybe it should be CustomResourceDefinition unless we mean an instance of a custom resource like it is in the kube docs, I just hate things being formatted this way in docs :sob: does it make sense to change it to CustomResourceDefinition though?

Reply:
Imma say "CustomResourceDefinition object" everywhere I think for this...

Reply:
hah, no, I'm convinced by the argument that the Kube docs say the CR is the thing installed by the CRD and that this is, in fact, a total bike shed. Let's go with your simpler version :D.

Reply:
So...which one? You know more than me so if you make a call I'm happy to go with it :) sorry just not sure which is the "simpler version you mean" :joy:

Reply:
I personally think, reflecting on this and the fact it seems like a hellmouth to the bike-shed dimension, we should just go with the one with less words. So I vote "custom resources". If anyone _really_ cares that this is technically wrong (Im not even sure it is now, honestly 😅) we can always change it. I regret bringing it up 😂.